### PR TITLE
genymotion-saas-install-gmsaas-cli 0.0.1

### PR DESCRIPTION
**Pull request by thomascarpentier**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/genymotion-saas-install-gmsaas-cli/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+title: Genymotion SaaS install gmsaas cli
+summary: |
+  Install gmsaas,  command line interface to rule all your virtual devices hosted in Genymotion SaaS solution.
```
Comment (ofalvai) [inline steps/genymotion-saas-install-gmsaas-cli/0.0.1/step.yml:3]:
```suggestion
  Install gmsaas, a command line interface to rule all your virtual devices hosted in Genymotion SaaS solution.
```

File: steps/genymotion-saas-install-gmsaas-cli/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+description: |
+  This step installs gmsaas cli, it allaows user to rule all your virtual devices hosted in Genymotion SaaS solution without opening an Internet browser.
```
Comment (ofalvai) [inline steps/genymotion-saas-install-gmsaas-cli/0.0.1/step.yml:5]:
```suggestion
  This step installs gmsaas cli, it allows tou to rule all your virtual devices hosted in Genymotion SaaS solution without opening an internet browser.
```

File: steps/genymotion-saas-install-gmsaas-cli/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+published_at: 2023-01-25T11:57:35.878313175+01:00
+source:
+  git: https://github.com/genymobile/bitrise-step-genymotion-saas-install-gmsaas-cli.git
+  commit: 2ca25eaf9c778cfe35d3a31bc072b24820ead71c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
```
Comment (ofalvai) [inline steps/genymotion-saas-install-gmsaas-cli/0.0.1/step.yml:18]: nit: Xamarin platform is now deprecated, feel free to remove it from this list

**Review by ofalvai — CHANGES_REQUESTED** [review]
Just a few typos, looks good otherwise!

**Review by thomascarpentier — COMMENTED** [review]
Remove Xamarin platform as is now deprecated

File: steps/genymotion-saas-install-gmsaas-cli/0.0.1/step.yml (hunk 3 above)
Comment (tcarpentier) [reply steps/genymotion-saas-install-gmsaas-cli/0.0.1/step.yml]: Hi @ofalvai, thanks for the review. I've updated all files.

**Review by ofalvai — APPROVED** [review]

**Pull request merged**